packages/atomic/src/components/search/result-lists/result-list-common.tsx

```diff
@@ -84,6 +84,7 @@ export interface RenderListOptions extends DisplayOptions {
   getContentOfResultTemplate(
     result: Result | FoldedResult
   ): HTMLElement | DocumentFragment;
+  ready: boolean;
```

Comment:
It looks like `ready` is set  in the `onReady` callback anyway, so perhaps we should handle it within `result-list-common`

Reply:
ready is a @State property in the components, state property change trigger re-renders, a private property in the list common wouldnt